Gems/EditorPythonBindings/Code/Source/PythonSystemComponent.cpp

```diff
@@ -237,8 +237,8 @@ namespace EditorPythonBindings
             {
                 ec->Class<PythonSystemComponent>("PythonSystemComponent", "The Python interpreter")
                     ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
-                        ->Attribute(AZ::Edit::Attributes::AppearsInAddComponentMenu, AZ_CRC_CE("System"))
-                        ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
+                    ->Attribute(AZ::Edit::Attributes::AppearsInAddComponentMenu, AZ_CRC_CE("System"))
```

Comment:
Is this styling change intentional, or did visual studio do this to you? It's a common practice to tab in attributes on class elements.

Reply:
this was the .clang format thing automatically changing things, I think I added a few lines at the end